src/style.css

```diff
@@ -1,3 +1,102 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  background-color: grey;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  line-height: 60px;
+  height: 60px;
+  margin-right: 50px;
```

Comment:
The `margin-right` in the `.nav` class contradicts the requirement for the header to stretch the full page width without horizontal margins. Consider removing or adjusting this margin to meet the layout requirements.